c7n/policy.py

```diff
@@ -410,8 +410,8 @@ def resolve_resources(self, event):
         mode = self.policy.data.get('mode', {})
         resource_ids = CloudWatchEvents.get_ids(event, mode)
         if resource_ids is None:
-            raise ValueError("Unknown push event mode %s", self.data)
-        self.policy.log.info('Found resource ids: %s', resource_ids)
+            raise ValueError("Unknown push event mode%s", self.data)
+        self.policy.log.info('Found resource ids:%s', resource_ids)
```

Comment:
the parsing logic you have for k:v won't work for this.

Reply:
so the format here would be ids:['xyz', 'abc', 'def'] it looks like the parse would pickup {'ids': '['abc'"}

Reply:
yeah i wasn't sure how to handle that. i was thinking of using `ast.literal_eval` . Allows for lists and dicts to be parsed correctly. 

https://docs.python.org/3/library/ast.html#ast.literal_eval